api/db-cdr/csv/achilles_results2.csv

```diff
@@ -0,0 +1,2 @@
+"id","analysis_id","stratum_1","stratum_2","stratum_3","stratum_4","stratum_5","count_value"
```

Comment:
For somereason we could not figure out , liquibase would fail and get a mysql syntax error when this last row was in the csv. After a lot of experimenting it appears to be a weird liquibase bug. The file imports fine in every other mysql tool.  So we moved this last line to a new file. 

Should we just delete this file? We don't really need this line for testing. It is only the count of French  nationality.